junit-jupiter-params/src/test/java/org/junit/jupiter/params/ParameterizedTestIntegrationTests.java

```diff
@@ -695,6 +705,32 @@ void testWithAggregator(@AggregateWith(StringAggregator.class) String concatenat
 			fail("concatenation: " + concatenation);
 		}
 
+		@ParameterizedTest
+		@CsvSource(value = { " ab,  cd", "ef, gh" }, ignoreTrailingAndLeadingWhitespace = false)
+		void testCsvSourceWithIgnoreTrailingAndLeadingWhitespaceSetToFalse(String argument) {
+			fail("arg:" + argument);
+		}
+
+		@ParameterizedTest
+		@CsvSource(value = { " ab,  cd", "ef, gh" }, ignoreTrailingAndLeadingWhitespace = true)
+		void testCsvSourceWithIgnoreTrailingAndLeadingWhitespaceSetToTrue(String argument) {
+			fail("arg:" + argument);
+		}
+
+		@ParameterizedTest
+		@CsvFileSource(resources = "/trailing-leading-spaces.csv",
+				ignoreTrailingAndLeadingWhitespace = false)
+		void testCsvFileSourceWithIgnoreTrailingAndLeadingWhitespaceSetToFalse(String argument) {
+			fail("arg:" + argument);
+		}
+
+		@ParameterizedTest
+		@CsvFileSource(resources = "/trailing-leading-spaces.csv",
+				ignoreTrailingAndLeadingWhitespace = true)
+		void testCsvFileSourceWithIgnoreTrailingAndLeadingWhitespaceSetToTrue(String argument) {
+			fail("arg:" + argument);
+		}
```

Comment:
Please also add corresponding tests that check these two methods.

Reply:
done, sorry missed that.